PayPal/src/test/java/com/braintreepayments/api/PayPalClientUnitTest.java

```diff
@@ -240,6 +241,34 @@ public void tokenizePayPalAccount_startsBrowser() throws JSONException, BrowserS
         assertEquals("paypal-browser", metadata.get("source"));
     }
 
+    @Test
+    public void tokenizePayPalAccount_whenDefaultDeepLinkHandlerEnabled_startsBrowserAsSingleTask() throws JSONException, BrowserSwitchException {
```

Comment:
Same comment throughout re: internal naming

```suggestion
    public void tokenizePayPalAccount_whenLaunchesBrowserSwitchAsNewTaskEnabled_startsBrowserAsSingleTask() throws JSONException, BrowserSwitchException {
```

Reply:
We could also 🏈 these internal changes to a separate PR after the release too